crates/sui-core/src/consensus_handler.rs

```diff
@@ -504,12 +505,16 @@ impl MysticetiConsensusHandler {
     pub fn new(
         mut consensus_handler: ConsensusHandler<CheckpointService>,
         mut receiver: UnboundedReceiver<consensus_core::CommittedSubDag>,
+        commit_consumer_monitor: Arc<CommitConsumerMonitor>,
     ) -> Self {
         let handle = spawn_monitored_task!(async move {
+            // TODO: pause when execution is overloaded, so consensus can detect the backpressure.
```

Comment:
This todo is for handling backpressure in normal commit operations right?

Reply:
This is for both "normal" and catchup operations, but mostly for catchup.